doc/migration/v0.1.0-migration-guide.md

```diff
@@ -0,0 +1,268 @@
+
+    // Watch for changes to the secondary resource AppService and enqueue reconcile requests for the owner Memcached
+    err = c.Watch(&source.Kind{Type: &appv1alpha1.AppService{}}, &handler.EnqueueRequestForOwner{
+		IsController: true,
+		OwnerType:    &cachev1alpha1.Memcached{},
+	})
+    ```
+-  Add a new controller to watch and reconcile the CR independently of the other CR.
+    ```sh
+    $ operator-sdk add controller --api-version=app.example.com/v1alpha1 --kind=AppService
+    ```
+    ```Go
+    // Watch for changes to the primary resource AppService
+    err = c.Watch(&source.Kind{Type: &appv1alpha1.AppService{}}, &handler.EnqueueRequestForObject{})
+    ```
+
+### Copy and modify reconcile code from pkg/stub/handler.go
+
+In a `v0.1.0` project the reconcile code is defined in the `Reconcile()` method of a controller's [Reconciler][reconciler-go-doc]. This is similar to the `Handle()` function in the older project. Note the difference in the arguments and return values:
+- `func (r *ReconcileMemcached) Reconcile(request reconcile.Request) (reconcile.Result, error)`
+- `func (h *Handler) Handle(ctx context.Context, event sdk.Event) error`
+
+Instead of receiving an sdk.Event (with the object), the Reconcile function receives a [Request][request-go-doc] (Name/Namespace key) to lookup the object. If an error is returned the Request will be retried. Otherwise the [Result][result-go-doc] can be explicitly configured to retry immediately or periodically after a timeout.
+
+Copy the code from the old project's `Handle()` function over the existing code in your controller's `Reconcile()` function.
+Be sure to keep the initial section in the `Reconcile()` code that looks up the object for the Request and checks to see if it's deleted.
+
+```Go
+import (
+    apierrors "k8s.io/apimachinery/pkg/api/errors"
+    cachev1alpha1 "github.com/example-inc/memcached-operator/pkg/apis/cache/v1alpha1"
+    ...
+)
+func (r *ReconcileMemcached) Reconcile(request reconcile.Request) (reconcile.Result, error) {
+	// Fetch the Memcached instance
+	instance := &cachev1alpha1.Memcached{}
+	err := r.client.Get(context.TODO(), request.NamespacedName, instance)
+	if err != nil {
+		if apierrors.IsNotFound(err) {
+			// Request object not found, could have been deleted after reconcile request.
+			// Owned objects are automatically garbage collected.
+			// Return and don't requeue
+			return reconcile.Result{}, nil
+		}
+		// Error reading the object - requeue the request.
+		return reconcile.Result{}, err
+    }
+
+    // Rest of your reconcile code goes here.
+    ...
+}
+```
+
+Change the following in your reconcile code:
+- Replace `return err` with `return reconcile.Result{}, err`
+- Replace the calls to the SDK client(Create, Update, Delete, Get, List) with the reconciler's client
+
+See the examples below and the controller-runtime [client API doc][client-api-doc] for more details.
+```Go
+// Create
```

Comment:
Just a suggestion, but for easier reading maybe a table would be better:

|        | v0.0.x                                                          | v.0.1                                                 |
|--------|-----------------------------------------------------------------|-------------------------------------------------------|
| Create | ```Go dep := &appsv1.Deployment{...} err := sdk.Create(dep) ``` | ```Go err := r.client.Create(context.TODO(), dep) ``` |
|   ...     |                              ...                                   |                       ...                                |
|  ...      |                         ...                |                                ...                       |

Reply:
@LiliC I've formatted it as a table and split the code into multi-line rows, but the lines wrap around a bit when I view the full doc. Not sure if that makes it as clear. Plus I can't seem to format the single lines as Go code. Seems like it has to be a block. You can see that in your comment above as well.

I've left both the table and code block in for comparison so you can see.
Also a screenshot of what I'm seeing:
![image](https://user-images.githubusercontent.com/6660644/47385697-99aab800-d6bf-11e8-94b8-790f947c491c.png)

Reply:
I'll wait to see if we can't format the table a bit better. Else I'm leaning towards keeping the code block which is a little more readable.

Reply:
Makes sense 👍